composer.json

```diff
@@ -7,6 +7,7 @@
   "bin": [
     "bin/terminus.bat", "bin/terminus"
   ],
+  "minimum-stability": "beta",
```

Comment:
At the moment, Robo dev-master is pulling in the 2.0.0-beta1 releases of annotated commands and output formatter libraries for features I added for Terminus.  Robo 1.0.0 stable is due out soon; I will make the 2.0.0 stable releases for annotated commands and output formatters before then, at which point we can remove this again.